test/services/school/index.test.js

```diff
@@ -283,72 +266,78 @@ describe('school service', () => {
-			expect(result.isTeamCreationByStudentsEnabled).to.be.false;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'opt-in');
-			result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.true;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'opt-out');
-			result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.true;
-		});
-
-		it('team creation by students should be updated according to '
-			+ 'environment setting and disabled by admin', async () => {
-			// school with enabled student team creation by admin
-			const school = await testObjects.createTestSchool({ enableStudentTeamCreation: false });
-			expect(school.enableStudentTeamCreation).to.be.false;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'enabled');
-			let result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.true;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'disabled');
-			result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.false;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'opt-in');
-			result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.false;
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'opt-out');
-			result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.false;
-		});
+				Configuration.set('STUDENT_TEAM_CREATION', 'opt-out');
+				result = await app.service('/schools').get(school._id);
+				expect(result.isTeamCreationByStudentsEnabled).to.be.true;
+			}
+		);
+
+		it(
+			'team creation by students should be updated according to ' + 'environment setting and enabled by admin',
+			async () => {
+				// school with enabled student team creation by admin
+				const school = await testObjects.createTestSchool({ enableStudentTeamCreation: true });
+				expect(school.enableStudentTeamCreation).to.be.true;
+
+				Configuration.set('STUDENT_TEAM_CREATION', 'enabled');
+				let result = await app.service('/schools').get(school._id);
+				expect(result.isTeamCreationByStudentsEnabled).to.be.true;
+
+				Configuration.set('STUDENT_TEAM_CREATION', 'disabled');
+				result = await app.service('/schools').get(school._id);
+				expect(result.isTeamCreationByStudentsEnabled).to.be.false;
+
+				Configuration.set('STUDENT_TEAM_CREATION', 'opt-in');
+				result = await app.service('/schools').get(school._id);
+				expect(result.isTeamCreationByStudentsEnabled).to.be.true;
+
+				Configuration.set('STUDENT_TEAM_CREATION', 'opt-out');
+				result = await app.service('/schools').get(school._id);
+				expect(result.isTeamCreationByStudentsEnabled).to.be.true;
+			}
+		);
+
+		it(
+			'team creation by students should be updated according to ' + 'environment setting and disabled by admin',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)